go/runtime/flow_consumer.go

```diff
@@ -297,7 +298,12 @@ func shardApply(ctx context.Context, svc *consumer.Service,
 		if nextSpec != nil && logSpec == nil {
 			// Grab labeled catalog task and its journal rules.
 			var name = nextSpec.LabelSet.ValueOf(labels.TaskName)
-			var _, commons, _, err = catalog.GetTask(name)
+			var revisionStr = nextSpec.LabelSet.ValueOf(labels.TaskCreated)
+			var minTaskRevision, err = strconv.ParseInt(revisionStr, 10, 64)
```

Comment:
nit: `taskCreated` instead of `minTaskRevision` ? Can keep the naming consistent. IMO the "revision" suffix isn't adding anything.